.github/workflows/sandbox-deleting.yml

```
@@ -24,43 +24,71 @@ jobs:
         run: |
-          RATE_LIMIT=$(curl -s -H "Authorization: Bearer $GITHUB_TOKEN" \
-            -H "Accept: application/vnd.github+json" \
-            "https://api.github.com/rate_limit")
-          REMAINING=$(echo "$RATE_LIMIT" | jq .rate.remaining)
-          if [ "$REMAINING" -lt 10 ]; then
-            echo "::warning::GitHub API rate limit is low: $REMAINING requests remaining"
+          if ! command -v jq &> /dev/null; then
+            echo "::error::jq is required but not installed"
+            exit 1
+          fi
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)